docs/samples/deployments/app-service/presidio-services.json

```diff
@@ -204,7 +277,7 @@
                         "value": "[parameters('imagesArePublic')]"
                     },
                     "image": {
-                        "value": "[parameters('anonymizerImage')]"
+                        "value": "[parameters('imageRedactorImage')]"
```

Comment:
seems that the imageRedactor deployment is done twice? there's a second image-redactor deployment later in the file

Reply:
fixed